src/core/index.js

```diff
@@ -19,7 +19,8 @@ var core = module.exports = Object.assign(require('./const'), require('./math'),
     Container:              require('./display/Container'),
     Transform:              require('./display/Transform'),
     TransformStatic:        require('./display/TransformStatic'),
-    TransformManual:        require('./display/TransformManual'),
+    TransformBase:          require('./display/TransformBase'),
+    TransformManual:        require('./display/TransformBase'),
```

Comment:
Yes, temporarily, for pixi-spine, its using TransformManual right now. I'll remove that as soon as i change pixi-spine to TransformBase.